### ISSUE TITLE
Ability to specify USER for commands?

### ISSUE THREAD

**Issue by andyshinn** [rhuss/docker-maven-plugin#327]
I see that there is a user setting for owner of files copied by assembly. But not one for running `CMD` or `ENTRYPOINT`. Is there a way to specify the `USER` so that when I run the container outside of Maven it is using the correct user?

**Comment by Khazrak** [comment]
I also have this problem. Or for example if you want to switch to root to install more packages and then switch back to non-root user

**Comment by jgangemi** [comment]
no, not at this time, however, you can write and specify your own `Dockerfile` to achieve this.

**Comment by rhuss** [comment]
This is currently not planned, but could be a good addition. Maybe using the same specification as for adding file, so the `<runCmds>` section would need to be added to allow the specification of a user.

But for the momemnt, @jgangemi is right, a Dockerfile is probably better suited here. We will extend for the Dockerfile mode shortly to make it more powerful (e.g. allowing to add the assembly defined in the pom.xml in a plain Dockerfile). See #205 as followup.
